enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.tsx

```diff
@@ -65,7 +65,7 @@ export const _InteractiveQuestion = ({
   const hasQuestionChanges =
     card && (!card.id || card.id !== card.original_card_id);
 
-  const [loading, setLoading] = useState(true);
+  const [isQuestionLoading, setIsQuestionLoading] = useState(true);
 
   const { isRunning } = uiControls;
```

Comment:
I wonder if we should also alias `isRunning` to `isQueryRunning` too, so we can distinguish between question and query.

Reply:
Good idea.

Reply:
Fixed in 4654ee28410d938452c7a5127e7861388f78a6f2